internal/pipeline/steps.go

```diff
@@ -137,18 +137,23 @@ func stepByKeyInference(o *ordered.MapSA) (Step, error) {
 // sign adds signatures to each command step (and recursively to any command
 // steps that are within group steps. The steps are mutated directly, so an
 // error part-way through may leave some steps un-signed.
-func (s Steps) sign(env map[string]string, key jwk.Key) error {
+func (s Steps) sign(key jwk.Key, env map[string]string, pInv *PipelineInvariants) error {
```

Comment:
👍

Reply:
I feel like there's a another place that's like this. I'll see if I can put the key consistently as the first arg everywhere.